### PR TITLE
Add guidelines for starting new localizations

### PR DESCRIPTION
**Pull request by svrnm**
This PR updates the contribution guidelines for localizations to include instructions how to start a new localization project.

### CODE REVIEW

File: content/en/docs/contributing/localization.md [hunk 1]
```diff
@@ -204,22 +204,48 @@ For more details about the script, run `npm run check:i18n -- -h`.
...
+
+You don't have to be an existing contributor to the OpenTelemetry project, to
+start a new localization. However you will not be added as an member of the
+[OpenTelemetry GitHub organization](https://github.com/open-telemetry/) and as a
+member of the approvers group for your localization. You will need to satisfy
+the requirements for becoming an established member and approver as outlined in
+the
+[membership guidelines](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md).
+
+While bootstraping the localization project, maintainers will treat your reviews
```
Comment (theletterf) [inline content/en/docs/contributing/localization.md:245]:
```suggestion
When starting the localization project, maintainers will treat your reviews
```

File: content/en/docs/contributing/localization.md [hunk 2]
```diff
@@ -204,22 +204,48 @@ For more details about the script, run `npm run check:i18n -- -h`.
...
 ## New localizations
 
-(Section To Be Completed soon with information about how to propose a new
-localization.)
+To start a new localization for the OpenTelemetry website,
+[raise an issue](https://github.com/open-telemetry/opentelemetry.io/issues/) to
+share your interest to contribute. Tag all other individuals, that are willing
+to write and review translations in the language you want to add. You need at
+least two potential contributors, three would be ideal. Include the following
+task list in your issue as well:
```
Comment (theletterf) [inline content/en/docs/contributing/localization.md:212]:
```suggestion
share your interest to contribute. Tag all other individuals that are willing
to write and review translations in the language you want to add. You need at
least two potential contributors, three would be ideal. Include the following
task list in your issue as well:
```

File: content/en/docs/contributing/localization.md [hunk 3]
```diff
@@ -204,22 +204,48 @@ For more details about the script, run `npm run check:i18n -- -h`.
...
+
+- List at least 2 contributors for the new language: `<@gitHubHandle1>`,
+  `<@gitHubHandle2>`, ...
+- Localize index page to `<your language>`
+- Create an issue label for `lang:<shortcode>`
+- Create org-level group for `<shortcode>` approvers
+- Update components owners for `content/<shortcode>`
```
Comment (theletterf) [inline content/en/docs/contributing/localization.md:219]: How about having this as a Markdown code snippet they could paste in the issue?

File: content/en/docs/contributing/localization.md [hunk 4]
```diff
@@ -204,22 +204,48 @@ For more details about the script, run `npm run check:i18n -- -h`.
...
+For the `<shortcode>` use the
+[ISO 639-1 code](https://en.wikipedia.org/wiki/ISO_639-1) for the language you
+want to add.
+
+After you created that issue and have the required amount of contributors,
+maintainers will ask you to provide a pull request with a translation of the
+[index page](https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/_index.md).
+Make sure that maintainers are allowed to edit your PR, since they will add
+additional changes to your PR, that are required to get your localization
+project started.
```
Comment (theletterf) [inline content/en/docs/contributing/localization.md:230]:
```suggestion
additional changes to your PR that are required to get your localization
project started.
```

File: content/en/docs/contributing/localization.md [hunk 5]
```diff
@@ -204,22 +204,48 @@ For more details about the script, run `npm run check:i18n -- -h`.
...
+
+With your first PR merged maintainers will take care of setting up the issue
+label, the org-level group and the component owners.
+
+{{% alert title="Important" color="warning" %}}
+
+You don't have to be an existing contributor to the OpenTelemetry project, to
+start a new localization. However you will not be added as an member of the
```
Comment (theletterf) [inline content/en/docs/contributing/localization.md:238]:
```suggestion
start a new localization. However you will not be added as a member of the
```

File: content/en/docs/contributing/localization.md [hunk 6]
```diff
@@ -204,22 +204,48 @@ For more details about the script, run `npm run check:i18n -- -h`.
...
+[OpenTelemetry GitHub organization](https://github.com/open-telemetry/) and as a
```
Comment (theletterf) [inline content/en/docs/contributing/localization.md:239]:
```suggestion
[OpenTelemetry GitHub organization](https://github.com/open-telemetry/) or as a
```

**Review by theletterf — APPROVED** [review]
Great! Some suggestions.

File: content/en/docs/contributing/localization.md (hunk 3 above)
Comment (svrnm) [reply content/en/docs/contributing/localization.md:219]:
~~~suggestion
```
- [ ] List at least 2 contributors for the new language: `<@gitHubHandle1>`,
  `<@gitHubHandle2>`, ...
- [ ] Localize index page to `<your language>`
- [ ] Create an issue label for `lang:<shortcode>`
- [ ] Create org-level group for `<shortcode>` approvers
- [ ] Update components owners for `content/<shortcode>`
```
~~~

Comment (svrnm) [reply content/en/docs/contributing/localization.md]: done

**Pull request merged**